pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -654,6 +598,172 @@ func TestUserSignupWithExistingMUROK(t *testing.T) {
 	require.Equal(t, v1.ConditionTrue, cond.Status)
 }
```

Comment:
Don't forget to verify that the `Status.CompliantUsername` is properly set
```suggestion
	assert.Equal(t, mur.Name, instance.Status.CompliantUsername)
}
```

Reply:
Done.